### PR TITLE
Workaround problem with 0 dim in shape

### PR DESCRIPTION
**Pull request by tsocha**
#53

### CODE REVIEW

**Comment by kuke** [comment]
LGTM

File: fluid_onnx/variables.py [hunk 1]
```diff
@@ -22,6 +22,8 @@ def paddle_variable_to_onnx_tensor(paddle_var_name, block):
     # TODO(varunarora): Need to do this only in the case of VarType.LOD_TENSOR.
     paddle_var = block.var(paddle_var_name)
     shape = paddle_onnx_shape(paddle_var.shape)
+    if shape[0] == 0:
```
Comment (kuke) [inline fluid_onnx/variables.py]: Can remove these two lines

Comment (tsocha) [reply fluid_onnx/variables.py]: Agreed.

**Review by kuke — APPROVED** [review]
LGTM

**Pull request merged**